src/Microsoft.CodeQuality.Analyzers/VisualBasic/Maintainability/BasicRemoveUnusedLocals.vb

```diff
@@ -57,14 +57,14 @@ Namespace Microsoft.CodeQuality.VisualBasic.Analyzers.Maintainability
 
                         operationBlockContext.RegisterOperationAction(
                         Sub(operationContext)
-                            Dim declarations = DirectCast(operationContext.Operation, IVariableDeclarationsOperation).Declarations
+                            Dim declarations = DirectCast(operationContext.Operation, IVariableDeclarationGroupOperation).Declarations
```

Comment:
We can simplify this with [https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Operations/OperationExtensions.cs#L102](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Operations/OperationExtensions.cs#L102)

Reply:
We don't even have to have the nested for loops, the extension I linked operates on IVariableDeclarationGroupOperation.

---
In reply to: [150135540](https://github.com/dotnet/roslyn-analyzers/pull/1346#discussion_r150135540) [](ancestors = 150135540)